tests/testthat/test-subset-access.R

```diff
@@ -83,8 +83,8 @@ test_colpaths <- function(tt) {
     cdf2 <- make_col_df(tt, visible_only = FALSE)
     res3 <- lapply(cdf$path, function(pth) rtables:::subset_cols(tt, pth))
     res4 <- lapply(cdf2$path, function(pth) rtables:::subset_cols(tt, pth))
-    expect_identical(res3, res4[!is.na(cdf2$abs_pos)])
-    expect_identical(res3, lapply(seq_len(ncol(tt)),
+    testthat::expect_identical(res3, res4[!is.na(cdf2$abs_pos)])
```

Comment:
There's no need to use `::` here (or nearly anywhere in package code), so please remove it here and elsewhere.

Reply:
I added it only because if the package is not loaded, `lintr` complains. Removing it now. There are many other cases where `::` and `:::` are present (the latter should not pass the `--as-cran` tests, is the automation using it?). Should I remove them all? It could be better to add an issue for this

Reply:
Dont worry about that, i'll take care of it. The rule is we need `::` whenever we are utilizing a suggests package, rather than a hard dependency (and those calls must come after a call to `requireNamespace(<suggested pkg>)`)

Currently, that is `flextable`, used (unsurprisingly) in `tt_to_flextable`. Everywhere else its not necessary and my next commit will remove it there.

Reply:
Ok, I see, I added it as a rule ;)